src/main/java/frc/robot/subsystems/BallIndexer.java

```diff
@@ -30,30 +31,33 @@
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-private AnalogInput indexAcquiredSensor;
-private AnalogInput indexSensor;
-private AnalogInput indexToShootSensor;
 private WPI_TalonSRX beltMotor;
+private DigitalInput indexSensor;
+private DigitalInput indexToShootSensor;
+private DigitalInput indexAcquiredSensor;
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
     private boolean indexerRunning = false;
 
     public BallIndexer() {
```

Comment:
Need to make sure wired correctly as the order changed because port 0 was already inn use.